Solutions/SOC Handbook/Workbooks/MITREAttack.json

```diff
@@ -61,7 +61,7 @@
+        "json": "# Getting Started\r\n---\r\n\r\nThis workbook enables SecOps Analysts, Threat Intelligence Professionals, and Threat Hunters to map out-of-the box Microsoft Sentinel detections coverage across MITRE ATT&CK framework. \r\n<br>With this workbook, analysts will be able to get insights into what out-of-the box detections are already available and can make informed decision to either onboard new datasources or enable new analytics and hunting queries to increase MITRE ATT&CK coverage or fill existing gaps. \r\n\r\nThe workbook is designed into separate tabs as described below.\r\n\r\n## DataSource Statistics\r\n---\r\nThis section provides dataSources currently being populated based on Usage Table and security alerts triggered across those datasources from SecurityAlerts Table across Microsoft Sentinel workspace.\r\n\r\n## Microsoft Sentinel Github\r\n---\r\nThis section provides breakdown of existing out-of-the box content by Detection type such as Analytics, Hunting Queries and Fusion detections as well as by MITRE ATT&CK Tactics. \r\n\r\n## Detection Platform Services\r\n---\r\nThis section provides breakdown of built-in alerts from Microsoft suite of security products such as Azure Defender for various cloud resource types/services, Microsoft Defender for Identity (MDI),  Microsoft Cloud Application Security (MCAS) and others. \r\n<br> These alerts can be visually represented by products as well as cloud resource types. \r\n\r\n## MITRE ATT&CK Heatmaps\r\n---\r\nThis section provides visual Heatmaps of different MITRE ATT&CK matrix categorized by cloud Platform such as Azure, AWS, GCP as well as by Microsoft Entra ID, Office 365, Windows, Linux.\r\n\r\nIn order to populate the data you can use either the Jupyter Notebook or deploy the Azure Function as instructed in the solution [ThreatAnalysis&ResponsewithMITREATT&CK](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/ThreatAnalysis%26Response)\r\n\r\n\r\nRefer Notebook : [![nbviewer](https://raw.githubusercontent.com/jupyter/design/master/logos/Badges/nbviewer_badge.svg)](https://nbviewer.jupyter.org/github/Azure/Azure-Sentinel/blob/master/Tools/MITREATT%26CK-LayerGeneration-Notebook/MITRE%20ATT%26CK%20for%20Microsoft%20Sentinel.ipynb)\r\n\r\n\r\n"
       },
```

Comment:
In "MITRE ATT&CK Heatmaps", section, remove everything and just add the same info message for MITRE ATT&CK Coverage built in feature as below.